src/components/EmailSubscription/EmailSubscription.scss

```diff
@@ -1,3 +1,45 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
 
-/* Delete these lines when you begin development */
+.EmailSubscription {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 100%;
+
+  &__searchBox {
```

Comment:
I'm seeing this border on focus. Should be able to get rid of it with something like

```scss
&:focus {
  outline: none;
}
```
![image](https://user-images.githubusercontent.com/20177171/97243283-332b6980-17b3-11eb-9e62-fcaf37f22598.png)

Reply:
I wouldn't remove input outlines; they're an accessibility feature https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

Reply:
The Figma design calls for no outline on the input box. What's the call here?

Reply:
@mocklee Bump!